apps/src/code-studio/levels/freeResponse.js

```diff
@@ -4,8 +4,9 @@ import { onAnswerChanged } from './codeStudioLevels';
 import { sourceForLevel } from '../clientState';
 
 export default class FreeResponse {
-  constructor(levelId) {
+  constructor(levelId, optional = false) {
```

Comment:
Nit: Default value `false` is sort of redundant since you coerce to boolean a couple of lines below.

Reply:
I must have forgotten what language I was writing in 🤷‍♂️